rubocop.yml

```diff
@@ -635,6 +635,7 @@ Style/MethodCallWithArgsParentheses:
   - puts
   Exclude:
   - "/**/Gemfile"
+  - "/**/db/migrate/*"
```

Comment:
Hmm, I am not sure if a blanket exclusion for migration files is the best thing to do. We want to exclude only because things like `create_table`, etc read better as DSLs, but people are not limited to writing those in migration files, and we probably **do** want to lint all other method calls.